### ISSUE TITLE
ZxingScannerComponent is not releasing all resources - BrowserCodeReader still trying to decode after ngDestroy

### ISSUE THREAD

**Issue by RafalOddslife** [zxing-js/ngx-scanner#557]
**Describe the bug**
BrowserCodeReader is trying to decode even after destroying ZxingScannerComponent
Affected version: "@zxing/ngx-scanner": "^17.0.1"

**Code**
```html
<button (click)="scannerVisible = !scannerVisible">Toggle scanner</button>
<zxing-scanner *ngIf="scannerVisible"></zxing-scanner>
``` 

**To Reproduce**
1. Go to [simple repro in stackblitz](https://stackblitz.com/edit/stackblitz-starters-dx4r2h?file=src%2Fmain.ts)
2. Use toggle button to init and destroy zxing-scanner component (the more you do this, the bigger the leakage)
3. Open chrome devtools, open Performance tab and capture new recording
4. BrowserCodeReader is still trying to decode even after destroying component

**Expected behavior**
All resources released

**Screenshots**
![image](https://github.com/zxing-js/ngx-scanner/assets/44264706/6e71c6b2-e378-4bcc-8523-d0783670aebf)

**Comment by werthdavid** [comment]
Thanks! Created a PR, feel free to take a look, my tests were fine. I'll merge it tmrw if nobody filed an objection..

**Comment by RafalOddslife** [comment]
I've just checked and I'm unable to reproduce the error for my repro so it appears to be working correctly. Appreciate fast reaction!

**Comment by werthdavid** [comment]
Released in 17.0.2
